src/components/HorizontalScroller.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "styles/modules/all";
+
+.horizontalScrollWrap {
+    position: relative;
+
+    .controls {
+        position: absolute;
+        top: 40%;
+        left: 20px;
+        transform: translateY(-50%);
+        z-index: 1;
+        width: calc(100% - 40px);
+        display: flex;
+        justify-content: space-between;
+        pointer-events: none;
+        z-index: 6;
```

Comment:
Might need to make this above 100. In the nextjs project for the inline walkthroughs the buttons are getting covered by the walkthrough icon.

![Screenshot 2024-05-10 at 3 34 30 PM](https://github.com/tollbros/tollbrothers-ui/assets/15040013/91d419fd-fdbe-4b9e-97e6-b9219da00374)